### PR TITLE
Add openawsem

### PR DESCRIPTION
**Pull request by cabb99**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cabb99** [comment]
@conda-forge/help-python, this PR is ready for review

**Comment by cabb99** [comment]
@conda-forge/staged-recipes @conda-forge/python Ready for review.

File: recipes/openawsem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/npschafer/openawsem.git
+  url: https://github.com/npschafer/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/openawsem/meta.yaml:15]:
```suggestion
```

File: recipes/openawsem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{% set name = "openawsem" %}
+{% set version = "1.1.1" %}
+
+package:
+  name:  {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/npschafer/openawsem.git
```
Comment (ocefpaf) [inline recipes/openawsem/meta.yaml:9]:
```suggestion
```

File: recipes/openawsem/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  entry_points:
+  - awsem_create = openawsem.scripts.mm_create_project:main
+  - awsem_run = openawsem.scripts.mm_run:main
+  - awsem_analyze = openawsem.scripts.mm_analyze:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.8
+    - numpy
```
Comment (ocefpaf) [inline recipes/openawsem/meta.yaml:30]:
```suggestion
```

**Pull request merged**